docs/inference_protocols.md

```diff
@@ -111,3 +111,42 @@ endpoints for Triton. These endpoints use the C API to communicate
 with the core of Triton. The primary source files for the endpoints
 are [grpc_server.cc](../src/servers/grpc_server.cc) and
 [http_server.cc](../src/servers/http_server.cc).
+
+## Java API (pre-release)
+
+The Triton Inference Server uses [Java CPP](https://github.com/bytedeco/javacpp)
+to create bindings around Tritonserver to create Java API. It requires the user
+to link a pre-built library to their application using Maven.   
+
+The API is documented in 
+[tritonserver.java](https://github.com/bytedeco/javacpp-presets/blob/master/tritonserver/src/gen/java/org/bytedeco/tritonserver/global/tritonserver.java).
+Alternatively, the user can refer to the web version [API docs](http://bytedeco.org/javacpp-presets/tritonserver/apidocs/)
+generated from `tritonserver.java`.
+A simple example using the Java API can be found in
+[Samples folder](https://github.com/bytedeco/javacpp-presets/tree/master/tritonserver/samples)
+which includes `Simple.java` which is similar to `simple.cc`. Please refer to
```

Comment:
Could be good to add the link to simple.cc here (https://github.com/triton-inference-server/server/blob/main/src/servers/simple.cc)